vispy/app/backends/_qt.py

```diff
@@ -636,7 +645,8 @@ def initializeGL(self):
     def resizeGL(self, w, h):
         if self._vispy_canvas is None:
             return
-        self._vispy_canvas.events.resize(size=(w, h))
+        self._vispy_set_physical_size(w, h)
+        self._vispy_canvas.events.resize(size=(self.width(), self.height()), physical_size=(w, h))
```

Comment:
PEP8 this needs to be shorter, you can run `python make flake` from the `vispy` root to see the error. But otherwise tests pass, which is great.